assets/app/view/game/stock_market.rb

```diff
@@ -208,18 +218,7 @@ def render
         children.concat(grid)
 
         if @explain_colors
-          type_text = [
-            [:par, 'Par value'],
-            [:no_cert_limit, 'Corporation shares do not count towards cert limit'],
-            [:unlimited, 'Corporation shares can be held above 60%'],
-            [:multiple_buy, 'Can buy more than one share in the corporation per turn'],
-            [:close, 'Corporation closes'],
-            [:endgame, 'End game trigger'],
-            [:liquidation, 'Liquidation'],
-            [:acquisition, 'Acquisition'],
-            [:repar, 'Par value after bankruptcy'],
-            [:ignore_one_sale, 'Ignore first share sold when moving price'],
-          ]
+          type_text = @game.class::MARKET_TEXT
```

Comment:
Hooray! I support this refactor.

Reply:
I was tempted to remove all notion of colors from the stock market class, one day maybe.